FabNESO/tasks.py

```diff
@@ -579,3 +582,199 @@ def log_density(
+        data = f.read()
+    d = literal_eval(data)
+    vvuq_params = {}
+    parameter_variations = {}
+    for parameter, param_values in d["parameters"].items():
+        vvuq_params[parameter] = {
+            "type": "float",
+            "default": param_values[0],
+        }
+        vary_length = 3
+        if len(param_values) == vary_length:
+            # Make the chaospy distribution. Possibly to be able to edit this later?
+            parameter_variations[parameter] = cp.Uniform(
+                param_values[1], param_values[2]
+            )
+    output_columns = d["output_columns"]
+    return (vvuq_params, parameter_variations, output_columns)
+
+
+@fab.task
+@fab.load_plugin_env_vars("FabNESO")
+def single_run_vvuq(
+    config: str,
+    mesh_file: str = "mesh.xml",
+    solver: str = "Electrostatic2D3V",
+    processes: str | int = 4,
+    nodes: str | int = 1,
+    cpus_per_process: str | int = 1,
+    wall_time: str = "00:15:00",
+    neso_outfile: str = "Electrostatic2D3V_line_field_evaluations.h5part",
+) -> None:
+    """
+    Run a single instance of the VVUQ campaign.
+
+    Run an instance of the VVUQ campaign using configurations in the designated solver.
+    The conditions and mesh are, by design, the defaults. When the iteration is done,
+    copy the NESO output to the local directory for later decoding through VVUQ.
+
+
+    Args:
+        config: Directory with single run configuration information.
+
+    Keyword Args:
+        solver: Which NESO solver to use.
+        mesh_file_name: Name of mesh XML in configuration directory.
+        processes: Number of processes to run.
+        nodes: Number of nodes to run on. Only applicable when running on a multi-node
+            system.
+        cpus_per_process: Number of processing units to use per process. Only
+            applicable when running on a multi-node system.
+        wall_time: Maximum time to allow job to run for. Only applicable when submitting
+            to a job scheduler.
+        neso_outfile: Name of the output file the NESO solver produces, to copy across
+            to the local directory.
+
+    """
+    # If we're running remotely, tell the submission to wait until done
+    if "archer2" in fab.env.remote:
+        fab.update_environment(
+            {
+                "job_dispatch": "cd /work/$project/$project/$username ; sbatch --wait",
+            }
+        )
+
+    # Make a temporary directory to store the configs in - they can be deleted when the
+    # run is done
+    with TemporaryDirectory(dir=config) as temporary_config_dir:
+        config_path = Path(temporary_config_dir)
+
+        # Copy the mesh and conditions files here for FabSIM to read it properly
+        shutil.copyfile(mesh_file, config_path / Path(mesh_file).name)
+        shutil.copyfile("conditions.xml", config_path / "conditions.xml")
+
+        # Run an instance of NESO
+        neso(
+            temporary_config_dir,
```

Comment:
Passing `temporary_config_dir` as the `config` argument to `neso` works when running on localhost but as its an absolute path on local filesystem, it doesn't work when running on a remote system, for example ARCHER2, with we getting permission denied errors when FabSim tries to create the config directory in the remote file system using a file path from the local file system.